services/slam/builtin/builtin.go

```diff
@@ -710,6 +717,25 @@ func (slamSvc *builtIn) StopSLAMProcess() error {
 	return nil
 }
 
+func (slamSvc *builtIn) getLazyPNGImage(ctx context.Context, cam camera.Camera) ([]byte, error) {
+	// We will hint that we want a PNG.
+	// The Camera service server implementation in RDK respects this; others may not.
+	img, _, err := camera.ReadImage(
+		gostream.WithMIMETypeHint(ctx, utils.WithLazyMIMEType(utils.MimeTypePNG)), cam)
+	if err != nil {
+		return nil, err
+	}
+
+	lazyImg, ok := img.(*rimage.LazyEncodedImage)
+	if !ok {
+		return nil, errors.Errorf("expected lazily encoded image, got %T", lazyImg)
+	}
+	if lazyImg.MIMEType() != utils.MimeTypePNG {
+		return nil, errors.Errorf("expected mime type %v, got %v", utils.MimeTypePNG, lazyImg.MIMEType())
+	}
+	return lazyImg.RawData(), nil
+}
+
```

Comment:
release() needs to be called after being done using the image

Reply:
I created a separate ticket for this, since currently `release` doesn't do anything yet, so there's some time to change the code to implement this.